gpiozero/boards.py

```diff
@@ -2237,3 +2237,98 @@ def off(self):
         """
         self.buzzer.value = None
         super(JamHat, self).off()
+
+class Pibrella(CompositeOutputDevice):
+    """
+    Extends :class:`CompositeOutputDevice` for the `Cyntech/Pimoroni Pibrella`_ board.
+
+    There are 3 LEDs, one button, a tonal buzzer, 4 outputs (with LEDs) and 4 inputs. The pins are fixed.
+    Usage::
+
+        from gpiozero import Pibrella
+
+        pb = Pibrella()
+
+        pb.lights.on()
+        pb.button.wait_for_press()
+        pb.outputs.on()
+        pb.buzzer.play('A4')
+        pb.inputs.a.wait_for_press()
+        pb.off()
+
+    :param bool pwm:
+        If :data:`True`, construct :class: PWMLED instances to represent each
+        LED on the board. If :data:`False` (the default), construct regular
+        :class:`LED` instances.
+
+    :type pin_factory: Factory or None
+    :param pin_factory:
+        See :doc:`api_pins` for more information (this is an advanced feature
+        which most users can ignore).
+
+    .. _Pibrella: https://www.pibrella.com/
+
+    .. attribute:: lights, outputs
+
+        :class:`LEDBoard` instance representing the three LEDs  
+
+        .. attribute:: red, yellow, green
+
+            :class:`LED` or :class:`PWMLED` instances representing the red,
+            yellow, and green LEDs .
+
+    .. attribute:: outputs
+    
+        :class:`LEDBoard` instance representing the 4 Out headers on the Pibrella.  
+        The Out headers are connected via a ULN2008 to 5V and can drive motors/steppers
+
+        .. attribute:: e, f, g, h
+
+            :class:`LED` or :class:`PWMLED` instances representing the 4 outputs .
+        
+    .. attribute:: button
+
+        The red :class:`Button` objects on the Pibrella.
+
+    .. attribute:: inputs
+    
+        :class:`ButtonBoard` instance representing the 4 In headers on the Pibrella.  
```

Comment:
Similarly, these might be better as a CompositeDevice of DigitalInputDevices?

Reply:
As per my comment for the outputs.